iree/turbine/kernel/compiler/dispatch_codegen.py

```diff
@@ -168,10 +170,23 @@ def abi_type(binding: BindingDesc):
                     pipeline_attr = iree_codegen_d.DispatchLoweringPassPipelineAttr.get(
                         iree_codegen_d.DispatchLoweringPassPipeline.None_
                     )
+                    translation_config = None
+                    if len(llvm_configs) > 0:
```

Comment:
minor: you can write this is as if not llvm_configs:

Reply:
sg and done, although it is `if llvm_configs` since we are checking for not 0.

Reply:
thanks